lib/compress/index.js

```diff
@@ -2166,6 +2166,9 @@ def_optimize(AST_Call, function(self, compressor) {
     if (can_inline && has_annotation(self, _INLINE)) {
         set_flag(fn, SQUEEZED);
         fn = make_node(fn.CTOR === AST_Defun ? AST_Function : fn.CTOR, fn, fn);
+        if (!exp.definition().single_use) {
+            fn = fn.clone(true);
+        }
```

Comment:
```suggestion
        fn = fn.clone(true);
```

I think we can always call this -- it's probably safer to do so.